### ISSUE TITLE
How to detach variable from gradient computation in tachi-scope

### ISSUE THREAD

**Issue by wangsd01** [yuanming-hu/difftaichi#39]
I was thinking to detach a variable from gradient computation in ti.kernel.

E.g y = copy(x).detach()
     f(x) = 5x + 6y
I hope df/dx = 5, not 11.

How can I do the above process in py-taichi? Thank you!
